backend/import/do_import.py

```diff
@@ -28,6 +28,16 @@ def is_visible(a):
     return not ('visibility' in a and a['visibility']['user_key'] == SECRET)
 
 
+def get_root(ou):
+    """Get root of current org unit."""
+    parent = ou.json['parent']
+    if not parent:
+        return ou
+    else:
+        parent = mo_api.OrgUnit(parent['uuid'])
```

Comment:
Isn't `parent` a recursive structure?

Reply:
Ah, yes! I actually thought it was only one layer. This means that I can implement this without calling the server more than once.

Reply:
I changed this. @danchr , do you have any more comments?